include/GMSH.cc

```diff
@@ -74,53 +74,58 @@ void GMSH::output_vertices() {
   }
 }
 
+//! \brief    Generate initial stress of each point and store
+//! \details  Get ver_stress, hor_stress, and shear (not used), write output
 void GMSH::output_stresses() {
 
-  const std::string outputfilename = "stresscheck.txt";
-  std::fstream stresscheck;
-  stresscheck.open(outputfilename, std::ios::out);
-
-  if (stresscheck.is_open()) {
+  //! Declare temp value of horizontal and vertical stresses;
+  double ver_stress;
+  double hor_stress;
 
-    //! Iterate through vector and print
+  //! ycoord contains a vector of the y-coordinates (vertical) of each point
```

Comment:
Ow, that's really interesting. I though y is always vertical? Then we need if function here